analyze_openapi_changes.py

```
@@ -0,0 +1,180 @@
+        print(f"Error parsing OpenAI response: {e}")
+        return "Error"
+    except Exception as e:
+        print(f"Error in OpenAI classification: {e}")
+        # Fallback to basic classification based on text content
+
+        if "paths" in diff_text.lower() or "endpoint" in diff_text.lower():
+            return "API Change"
+        else:
+            return "Production Update"
+
+def analyze_changes(old_schema: Dict[str, Any], new_schema: Dict[str, Any]) -> Dict[str, Any]:
```

Ruff (0.8.2)

`analyze_changes` is too complex (15 > 10)

(C901)